tdrs-backend/tdpservice/search_indexes/static/search_indexes/js/filters.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * Persist changelist filters state (collapsed/expanded).
+ */
+'use strict';
```

Comment:
Do we need this change for the moment or can this be included in the PR that fixes 508?